clients/cmd/fluentd/lib/fluent/plugin/out_loki.rb

```diff
@@ -112,6 +115,22 @@ def configure(conf)
             @record_accessors[k] = record_accessor_create(v)
           end
         end
+        @structured_metadata_accessors = {}
+        conf.elements.select { |element| element.name == 'structured_metadata' }.each do |element|
+          element.each_pair do |k, v|
+            element.has_key?(k) # rubocop:disable Style/PreferredHashMethods #to suppress unread configuration warning
```

Comment:
sorry, not a fluentd expert here. what the call to `has_key?` without guarding anything on that?